graph/SchemaGraph.java

```diff
@@ -418,26 +414,6 @@ public boolean isModified() {
         return isModified;
     }
 
-    // TODO: verify
```

Comment:
We no longer do reference counting on the SchemaGraph. The db Cache is what gets reference-counted